.github/dependabot.yml

```
@@ -0,0 +1,12 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

```suggestion
      interval: weekly
    day: saturday
    time: "10:00"
  open-pull-requests-limit: 10
  target-branch: development
  versioning-strategy: increase
```

Doesn't need to run daily, and our workflow requires PR's to go into `development`, so dependabot will need to follow suit.